Lib/shutil.py

```diff
@@ -897,7 +897,7 @@ def _get_uid(name):
     return None
 
 def _make_tarball(base_name, base_dir, compress="gzip", verbose=0, dry_run=0,
-                  owner=None, group=None, logger=None):
+                  owner=None, group=None, logger=None, root_dir=None):
     """Create a (possibly compressed) tar file from all the files under
     'base_dir'.
```

Comment:
Can you document the new parameter?

Reply:
It is an internal function. Most of parameters are the same as in `make_archive` where they are documented. There is no separate documentation for this function, and I think it is not needed.